docs/streams/upgrade-guide.html

```diff
@@ -1735,73 +1735,38 @@ <h3 class="anchor-heading"><a id="streams_api_changes_0101" class="anchor-link">
         </thead>
         <tbody>
           <tr>
             <td>Kafka Streams API (rows)</td>
-            <td>0.10.0.x</td>
-            <td>0.10.1.x and 0.10.2.x</td>
-            <td>0.11.0.x and<br>1.0.x and<br>1.1.x and<br>2.0.x</td>
             <td>2.1.x and<br>2.2.x and<br>2.3.x and<br>2.4.x and<br>2.5.x and<br>2.6.x and<br>2.7.x and<br>2.8.x and<br>3.0.x and<br>3.1.x and<br>3.2.x and<br>3.3.x and<br>3.4.x and<br>3.5.x and<br>3.6.x and<br>3.7.x and<br>3.8.x and<br>3.9.x</td>
             <td>4.0.x</td>
           </tr>
           <tr>
-            <td>0.10.0.x</td>
+            <td>2.4.x and<br>2.5.x</td>
+            <td>compatible with exactly-once v2 turned off</td>
             <td>compatible</td>
-            <td>compatible</td>
-            <td>compatible</td>
-            <td>compatible</td>
-            <td></td>
           </tr>
           <tr>
-            <td>0.10.1.x and 0.10.2.x</td>
-            <td></td>
-            <td>compatible</td>
-            <td>compatible</td>
-            <td>compatible</td>
-            <td></td>
-          </tr>
-          <tr>
-            <td>0.11.0.x</td>
-            <td></td>
-            <td>compatible with exactly-once turned off<br>(requires broker version 0.11.0.x or higher)</td>
-            <td>compatible</td>
-            <td></td>
-          </tr>
-          <tr>
-            <td>1.0.x and<br>1.1.x and<br>2.0.x and<br>2.1.x and<br>2.2.0 and<br>2.2.0</td>
-            <td></td>
-            <td>compatible with exactly-once turned off<br>(requires broker version 0.11.0.x or higher);<br>requires message format 0.10 or higher;<br>message headers are not supported<br>(requires broker version 0.11.0.x or higher<br>with message format 0.11 or higher)</td>
-            <td>compatible; requires message format 0.10 or higher;<br>if message headers are used, message format 0.11<br>or higher required</td>
-            <td>compatible</td>
-            <td></td>
-          </tr>
-          <tr>
-            <td>2.2.1 and<br>2.3.x and<br>2.4.x and<br>2.5.x and<br>2.6.x and<br>2.7.x and<br>2.8.x and<br>3.0.x and<br>3.1.x and<br>3.2.x and<br>3.3.x and<br>3.4.x and<br>3.5.x and<br>3.6.x and<br>3.7.x and<br>3.8.x and<br>3.9.x</td>
-            <td></td>
-            <td></td>
-            <td>compatible; requires message format 0.11 or higher;<br>enabling exactly-once v2 requires 2.5.x or higher</td>
+            <td>2.6.x and<br>2.7.x and<br>2.8.x and<br>3.0.x and<br>3.1.x and<br>3.2.x and<br>3.3.x and<br>3.4.x and<br>3.5.x and<br>3.6.x and<br>3.7.x and<br>3.8.x and<br>3.9.x</td>
+            <td>compatible; enabling exactly-once v2 requires broker version 2.5.x or higher</td>
             <td>compatible</td>
-            <td></td>
           </tr>
           <tr>
             <td>4.0.x</td>
-            <td></td>
-            <td></td>
-            <td></td>
-            <td>compatible; enabling exactly-once v2 requires broker version 2.5.x or higher</td>
+            <td>compatible</td>
```

Comment:
Given that KS 4.0.x does not include EOSv1 any longer, we should also call this out? KS 4.0 with EOSv2 does require broker 2.5+ too.

Given that we don't have EOSv1 in KS 4.0 any longer, we could say "enabling exactly-once requires broker version 2.5.x or higher" (ie, drop the `v2` thing).

Or: we just merge 4.0 row with `2.6.x - 3.9.x` row?

Reply:
Sorry for late update. I merge 4.0 to `2.6.x - 3.9.x`, so we don't need to update the description. Thanks.

![Screenshot 2025-03-10 at 9 45 58 PM](https://github.com/user-attachments/assets/8bf4c0fb-ead7-42c4-ab38-5e5ce4c6555f)